source/fab/tasks/fortran.py

```diff
@@ -537,19 +537,21 @@ def run(self, artifacts: List[Artifact]) -> List[Artifact]:
                 if end_nature is not None:
                     if end_nature != exp[0]:
                         end_message = 'Expected end of {exp} "{name}" ' \
-                                      'but found {found}'
+                                      'but found {found}: {line}'
```

Comment:
This seems unrelated to stopping on error but it's such a small thing I don't see a need to remove it.

Reply:
Agreed, this PR about error messages.